src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+X Loyal Command made it to CentComm. [Red text]
+0 Head Revolutionaries made it to CentComm. [Red text]
+```
+
+```
+Crew Major Victory! [Green text]
+All Revolutionaries were killed or deconverted! [Red text]
+```
+
+## Conversion
+
+Conversion is how the Revolutionary antag faction grows, and is a *requirement* to obtain Revolutionary Major Victory. To convert a crewmember they must be interacted with by a Revolutionary over time, and once a critera is met they can be converted to the Revolution. Crew are also able to de-convert Revolutionaries. Conversion is not restricted to Headrevs, though they have access to certain tools that benefit them in this regard.
+
+To represent the convertable state a crewmember is in and the faction they belong to, there is now a hidden nummerical stat called Loyalty Health Points (LHP). 
+
+### Loyalty & Loyalty Damage
+
+All sentient crew have Conversion Health Points, which can normally range between 100 to -100 LHP. All crew, unless specified, begin at a full 100 LHP.
+
+- Revolutionaries are able to do specific actions that "deal damage" to a crewmember's LHP, gradually lowering it with varying effectiveness depending on how covert or obviously antagonistic the action is. 
+- If a crewmember gets below <**10** LHP, they enter a Convertable state.
+  - As long as the crewmember is not a Revolutionary, their LHP can not be reduced below 0. They will remain in the Convertable state until their LHP rises above **10** LHP again.
+  - While a crewmember is in the Convertable state, they can be converted into a Revolutionary via specific actions available to Revolutionaries.
+  - When converted to a Revolutionary, their LHP is set to **-25**.
+  - A successful conversion is silent for surrounding players, however the player who is converted gets a pop-up and the appropriate music stinger.
+- Crew are able to "heal" LHP via specific actions, deconverting Revolutionaries into normal crewmembers.
+  - If a Revolutionary's LHP is no longer in the negatives (i.e. at >=0 LHP), they instantly revert into a normal crewmember with **25** LHP and lose their antag status. *This is true for Headrevs as well.*
+  - Deconversion causes the person to fall to the floor for a second, as they remember their original allegiance.
+
+Headrevs are unique in that they have a LHP range between **-500** and 100 LHP. Headrevs begin at **-500** LHP when they are given the antag status. As a benefit of being a Headrev, they are able to visually see the current LHP of any crewmember, similar to a health HUD display. 
+
+The Captain is also unique in that they always have 100 LHP and can never go below this. This makes the Captain unconvertable to the Revolution, and they are the only Command on the station with this property. CentComm agents and other Admeme should be similarly unconvertable by default.
+
+Unlike normal damage, LHP can only be dealt/healed to an individual who is alive and not in crit. 
```

Comment:
This is the usual conscious requirement we see in a lot of design, so it would presumably cover "players who are not dead, in crit or asleep".